src/main/java/frc/robot/subsystems/swerve/swervecontainer/SwerveContainer.java

```diff
@@ -0,0 +1,15 @@
+package frc.robot.subsystems.swerve.swervecontainer;
+
+import frc.robot.subsystems.swerve.SwerveConstants;
+import frc.robot.subsystems.swerve.gyro.ISwerveGyro;
+import frc.robot.subsystems.swerve.modules.IModule;
+
+public abstract class SwerveContainer {
+
+    public SwerveContainer(SwerveFactory.SwerveContainerKey key){}
```

Comment:
????
why do you need this key?

Reply:
To make the swerve containers content accessible only from swerve factory.